src/components/network/geo-data.js

```diff
@@ -26,6 +26,11 @@ export default class GeoData {
 
     linePositionsById = new Map();
 
+    GeoData(other) {
```

Comment:
Is this used ? It looks like a C++ or Java constructor, but a javascript constructor is denoted by the keyword 'constructor' if I rememeber well.